doc/sending-gcc-patch.md

```diff
@@ -0,0 +1,26 @@
+This guide explains what to do to send a GCC patch for review.
+
+All the commands are supposed to be run in the folder where you cloned GCC.
+
+```bash
+./contrib/gcc-changelog/git_check_commit.py
```

Comment:
If you have multiple commits, this script can also take a rev-list as argument. I often use it like this:
`contrib/gcc-changelog/git_check_commit.py upstream/master..HEAD`

(assuming you have an upstream origin with a master branch that you branched from).

For a specific commit you can also use the commit's hash as argument:

`contrib/gcc-changelog/git_check_commit.py abdef78989`

Reply:
Thanks! Updating the PR then.